js/client-topbar.js

```diff
@@ -485,6 +488,10 @@
 			buf += '<hr />';
 			buf += '<p><strong>Chat</strong></p>';
 			buf += '<p><label class="optlabel"><input type="checkbox" name="notournaments"' + (Dex.prefs('notournaments') ? ' checked' : '') + ' /> Ignore tournaments</label></p>';
+			if (Object.keys(settings).length) {
```

Comment:
This is above and beyond, I just wanted it not to _crash_ on older servers. :p

Reply:
Why *not* be more robust? :)

Reply:
I've even gone further - I now should be robust to *future* server protocol changes (Zarel/Pokemon-Showdown#5424) as well!